readme.md

```diff
@@ -2,9 +2,23 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T1130491)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
+
+# DevExpress Blazor Components - How to Update CSS Styles to v22.2
+
+This document summarizes the most often cases when you should use our private CSS selectors to apply a style to an element.
+ 
+You can also press `Ctrl+F` and search for a private CSS selector that you used in a previous version. This helps you find a selector used in v22.1 or prior, and copy the new equivalent of this selector.
+
+If you did not manage to find your scenario in this document, you can inspect a component render and create a new CSS selector as described in the following articles: 
```

Comment:
What do you mean by "inspect a component render"? For example, do you mean "inspect how a component is rendered"?

Or are you talking about using the browser's DevTools to actually "inspect" the element itself?